src/main/java/edu/wpi/first/gradlerio/GradleRIOPlugin.java

```diff
@@ -56,6 +56,9 @@ interface Parameters extends FlowParameters {
         @Override
         public void execute(Parameters parameters) {
             Optional<Throwable> failure = parameters.getBuildResult().get().getFailure();
+            if (System.getProperty("user.dir").contains("OneDrive")) {
+                throw new Error("Cannot use project inside OneDrive");
```

Comment:
Create a custom exception derived from RuntimeException and throw that. Also include the full path the project is in in the error.

Reply:
Is this better?